src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.settings.targets

```diff
@@ -6,7 +6,7 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50' OR '$(TargetGroup)' == 'netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
I think what you did for product code change is correct. Here is for test code. Theoretically, tests should behave the same way regardless it's running for uap or uapaot, which is why the original code considers both netcore50 and netcore50aot as NET_NATIVE.